src/signals/settings/categories/components/IconInput/IconInput.tsx

```diff
@@ -123,20 +128,21 @@ export const IconInput = ({ formMethods, icon }: Props) => {
   return (
     <FieldGroup>
       <StyledHeading>Icoon</StyledHeading>
-
-      {/* Informatie */}
-
+      <Detail
+        header={'Het icoon wordt getoond op de openbare meldingenkaart.'}
+        content={'Zorg voor een circel en exporteer als SVG.'}
+      >
+        {ICONEXAMPLE}
+      </Detail>
       <Controller
         name="icon"
         control={formMethods.control}
         render={({ field: { name } }) => (
           <StyledDiv>
-            {fileDataURL ? (
+            {fileDataURL && (
               <StyledIcon size={32}>
                 <img width={32} height={32} alt="Icoon" src={fileDataURL} />
               </StyledIcon>
-            ) : (
-              <StyledSpan>Niet ingesteld</StyledSpan>
             )}
```

Comment:
we leave it?

Reply:
It's not in the code? 
![image](https://github.com/Amsterdam/signals-frontend/assets/25867841/a9b73120-9eac-4e61-a31b-30c5ecb66549)

Where are your seeing this, because I'm not.  -???-